src/routes/api/speed-[segmentid]-[timeStart]-[timeEnd].js

```diff
@@ -6,7 +6,7 @@ const limiter = rateLimit({
 });
 
 export async function get(request) {
-	const limit = parseInt(import.meta.env.VITE_API_LIMIT, 10) || 20; // number of requests per minute
+	const limit = parseInt(import.meta.env.VITE_API_LIMIT, 10) || 50; // number of requests per minute
```

Comment:
This possibly also needs to change in the vercel settings? The number of segments is already 19 and 1 traffic_snapshot call is done / segment.